libafl_bolts/src/minibsod.rs

```diff
@@ -907,6 +907,27 @@ pub fn generate_minibsod<W: Write>(
     write_minibsod(writer)
 }
 
+#[cfg(all(unix, target_pointer_width = "32"))]
+#[allow(clippy::non_ascii_literal, clippy::too_many_lines)]
+pub fn generate_minibsod<W: Write>(
+    writer: &mut BufWriter<W>,
+    signal: Signal,
+    _siginfo: &siginfo_t,
+    ucontext: Option<&ucontext_t>,
+) -> Result<(), std::io::Error> {
+    writeln!(writer, "{:━^100}", " CRASH ")?;
+    if let Some(uctx) = ucontext {
+        // write_crash(writer, signal, uctx)?;
```

Comment:
Shouldn't this at least write the signal?

Reply:
Also, instead of copying the whole function, maybe just make a block and hide that behind the cfg? Could be less confusing

Reply:
Resolved.